speech/getting-started/speech_recognition.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>Can we link to the audio file / license?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>done</p>